classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -326,6 +390,19 @@ function publishIssue($args, $request) {
 
 		$articleSearchIndex = null;
 		if (!$issue->getPublished()) {
+			$confirmationText = __('editor.issues.confirmPublish');
+			import('lib.pkp.controllers.grid.pubIds.form.PKPAssignPublicIdentifiersForm');
+			$formTemplate = $this->getAssignPublicIdentifiersFormTemplate();
+			$assignPublicIdentifiersForm = new PKPAssignPublicIdentifiersForm($formTemplate, $issue, 1, $confirmationText);
```

Comment:
Better to use `true` rather than `1` -- I get nervous when I see magic numbers :)